miso-web/src/main/webapp/scripts/form.js

```diff
@@ -479,7 +486,7 @@ FormUtils = (function ($) {
           if (item.id < 0) {
             item.id = null;
           }
```

Comment:
This bit needs to be removed for this fix to work. The temporary ID should not be removed